src/app/app.component.html

```diff
@@ -15,4 +15,4 @@
   <div #focusLast tabindex="-1" aria-label="Tab To Main"></div>
   <router-outlet (activate)="setFocus()"></router-outlet>
 </div>
-<app-footer></app-footer>
+<app-footer *ngIf="router.url!=='/ubs-admin'"></app-footer>
```

Comment:
hmm ... that's fine so far, but don't forget to clear it after we create the layouts

Reply:
of course